core/server/data/migrations/versions/3.22/02-settings-key-renames.js

```diff
@@ -0,0 +1,82 @@
+const logging = require('../../../../../shared/logging');
+
+const renameMapping = [{
+    from: 'default_locale',
+    to: 'lang'
+}, {
+    from: 'active_timezone',
+    to: 'timezone'
+}, {
+    from: 'ghost_head',
+    to: 'codeinjection_head'
+}, {
+    from: 'ghost_foot',
+    to: 'codeinjection_foot'
+}];
+
+module.exports = {
+    config: {
+        transaction: true
+    },
+
+    async up(options) {
+        await Promise.map(renameMapping, async (renameMap) => {
```

Comment:
NABD but just FYI - you can do a standard loop here, no need for bluebird stuff

Reply:
@allouis , so something like this would be valid?
```js
await renameMapping.map((renameMap) => {
    return await options
                .transacting('settings')
                .where('key', renameMap.from)
                .update({
                    key: renameMap.to
                });
});
```

Reply:
Yeah or even:
```js
for (const renameMap of renameMapping) {
  await options.transaction('settings').blah...
}
```

:relaxed:

Reply:
No biggie if you CBA to change it though!!

Reply:
I'll keep it as is for now, as things are tested and confirmed :grimacing: Will have it on my mind for future cases :wink:

Reply:
If you wanted to use normal map it would be like:
```js
await Promise.all(renameMapping.map(async (renameMap) => {
    return await options
                .transacting('settings')
                .where('key', renameMap.from)
                .update({
                    key: renameMap.to
                });
}));
```